src/backends/torch/torchlib.cc

```diff
@@ -178,6 +182,8 @@ namespace dd
 
   std::vector<Tensor> TorchModule::parameters()
   {
+    if (_graph)
+      return _graph->parameters();
     if (_native)
```

Comment:
Is an `else` needed ?

Reply:
no else because of return :)    this contradicts previsous comment, if we want to chain net for free, we need to cat parameters and not only send first one